client/i18n/locales/english/translations.json

```
@@ -106,7 +106,9 @@
     "donate-now": "Donate Now",
     "confirm-amount": "Confirm amount",
     "play-scene": "Press Play",
-    "closed-caption": "Closed caption"
+    "closed-caption": "Closed caption",
+    "share-on-bluesky": "Share on BlueSky",
+    "share-on-insta": "Share on Threads"
```

```suggestion
    "share-on-threads": "Share on Threads"
```

This'll take more changes than just this line, but we shouldn't refer to Threads as Insta(gram) anywhere in the code, or that'll just make a mess when we get to the point where we *want* to include Instagram links.